tools/src/tester/ResultLogger.ts

```diff
@@ -152,8 +152,9 @@ export class ConsoleResultLogger implements ResultLogger {
   }
 
   #maybe_shorten_error_message(message: string | undefined): string | undefined {
-    if (message === undefined || message.length <= 128 || this._verbose) return message
-    const part = message.split(',')[0]
+    const cut_at = 256
```

Comment:
Nit: This is better off as a configurable number in the constructor, esp when we refactor the test framework to be a standalone product.

Reply:
I'll leave it as is for now, but we can expose it as an option when someone actually wants to change this value programmatically.